Types/Module.cs

```diff
@@ -200,7 +221,13 @@ internal static bool UniConflicts(Module mod1, Module mod2)
             {
                 return false;
             }
-            return mod1.conflicts.Any(conflict => mod2.ProvidesList.Contains(conflict.name));
+            foreach (var conflict in mod1.conflicts)
+            {
+                if (!mod2.ProvidesList.Contains(conflict.name))
+                    continue;
+               if(conflict.version_within_bounds(mod2.version)) return true;
```

Comment:
Yes. 
It may of been clearer if I left it in Linq. 
```mod1.conflicts.Any(conflict => mod2.ProvidesList.Contains(conflict.name))```
is changed to 
```mod1.conflicts.Any(conflict => mod2.ProvidesList.Contains(conflict.name) && conflict.version_within_bounds(mod2.version))```